third_party/intel/backend/compiler.py

```diff
@@ -210,9 +210,7 @@ def add_stages(self, stages, options):
 
     @functools.lru_cache()
     def hash(self):
-        version = subprocess.check_output([_path_to_binary("spirv-dis")[0], "--version"])
-        if type(version) is bytes:
-            version = version.decode("utf-8")
+        version = subprocess.check_output([_path_to_binary("spirv-dis")[0], "--version"], text=True)
```

Comment:
We also probably want to remove end of line from version:
```suggestion
        version = subprocess.check_output([_path_to_binary("spirv-dis")[0], "--version"], text=True).strip()
```

Reply:
done.